src/dynsight/_internal/analysis/rdf.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from ovito.io import import_file
+from ovito.modifiers import (
+    CoordinationAnalysisModifier,
+    DeleteSelectedModifier,
+    ExpressionSelectionModifier,
+    LoadTrajectoryModifier,
+    TimeAveragingModifier,
+)
+from scipy.signal import find_peaks
+
+
+class RadialDistributionFunction:
+    """Object for computing and analyzing the Radial Distribution Function.
+
+    The Radial Distribution Function (RDF) describes how particle density
+    varies as a function of the distance.
+
+    Parameters:
+        trajectory:
+            List of file paths including trajectory and optional topology file.
+
+        pipeline:
```

Comment:
This is not a parameter, this is an attribute? This should not be accessible to the user, it is constant.